definitions/npm/react-bootstrap_v0.32.x/flow_v0.53.x-/react-bootstrap_v0.32.x.js

```diff
@@ -1,8 +1,7 @@
 declare module "react-bootstrap" {
-  import type { Node, Element, Component, Ref } from 'react';
+  import type { Node, Element, ElementType, Component, Ref } from 'react';
```

Comment:
Why import `ElementType` when you've switched all of them to `React$ElementType`?